### DISCUSSION TITLE
Get the amount of a given account/liability at a given date?

### DISCUSSION THREAD

**Discussion by Ezwen** [firefly-iii/firefly-iii#8356] [Firefly III]
The title says it all: how can I get the amount of a given account/liability at a given date from past history?

I can't find a simple way to do that, and by default the UI only shows _current_ amounts.

But I'm quite sure I'm missing something obvious :sweat_smile:

**Comment by JC5** [comment]
You can browse the account at /accounts/show/x, let me know if the chart shows you what you need.

**Comment by Ezwen** [comment]
Good point, I thought this way of doing was only giving me access to the balance at the level of each past week, but I see now that the chart can be at the level of a single past day if the chosen interval is small enough. 

Meanwhile I'm learning to play a bit with the "Reports" tool, which can also generate a list of the balance of all accounts at a given date − even better for my needs, in fact.

Thanks for the help!
